### PR TITLE
Update DG (Glossary & User stories)

### LINKED ISSUE #30: Update DG (User stories)
Update the target user profile and value proposition to match the project direction you have selected. Give a list of the user stories (and update/delete existing ones, if applicable), including priorities. This can include user stories considered but will not be included in the final product.

### LINKED ISSUE #33: Update DG (Glossary)
Define terms that are worth recording.

### PR DESCRIPTION
**Pull request by HmuuMyatMoe**
Update DG Glossary #33 
Add more user stories #30 

Close #33

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -592,38 +606,38 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```
Comment (LiuMC-SG) [inline docs/DeveloperGuide.md:609]: By the convention for markdown they gave, should leave as 1. for these pointer. Same for all the other ones.

**Review by LiuMC-SG — CHANGES_REQUESTED** [review]
Minor problem with convention of markdown numbering. Instead of `1.` `2.`, it should still just be `1.` `1.`.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (HmuuMyatMoe) [reply docs/DeveloperGuide.md]: Done

**Review by LiuMC-SG — APPROVED** [review]
LGTM

**Pull request merged**